rules/called-element.js

```diff
@@ -8,7 +8,9 @@ const {
 
 const { reportErrors } = require('./utils/reporter');
 
-module.exports = function() {
+const ignoreNonExecutableProcess = require('./utils/ignoreNonExecutableProcess');
```

Comment:
Proposal: Dedicated `utils` export:

```suggestion
const { ignoreNonExecutableProcess } = require('./utils');
```

Additional proposal: Rename to `skipInNonExecutableProcess`:

```suggestion
const { skipInNonExecutableProcess } = require('./utils');
```

Reply:
I'm fine with the renaming, but there are a lot of utils so a `utils/index.js` would look quite messy.

Reply:
It is the existing pattern, right?

If there is reason to re-structure nothing would break from the consumer point of view if we still import through `util/index`. Hence I'm proposing that as an improvement.

Reply:
I don't understand. So you want all utils to be re-exported from `utils/index.js`? What is the existing pattern? Restructure what?

Reply:
If we decided to introduce a `utils` entry point re-exporting such members from individual files is straight forward, at least in ES6:

```javascript
export * from './rules';
```